src/net/sourceforge/kolmafia/swingui/panel/DailyDeedsPanel.java

```diff
@@ -3704,14 +3708,16 @@ public DeckOfEveryCardDaily() {
     public void update() {
       boolean bm = KoLCharacter.inBadMoon();
       boolean kf = KoLCharacter.kingLiberated();
-      boolean have =
-          InventoryManager.getCount(ItemPool.DECK_OF_EVERY_CARD) > 0
-              || (KoLCharacter.inLegacyOfLoathing()
-                  && InventoryManager.getCount(ItemPool.REPLICA_DECK_OF_EVERY_CARD) > 0);
+      boolean haveNormal = InventoryManager.getCount(ItemPool.DECK_OF_EVERY_CARD) > 0;
+      boolean haveReplica =
+          KoLCharacter.inLegacyOfLoathing()
+              && InventoryManager.getCount(ItemPool.REPLICA_DECK_OF_EVERY_CARD) > 0;
       boolean nocards = Preferences.getInteger("_deckCardsDrawn") >= 15;
-      boolean allowed = StandardRequest.isAllowed(RestrictedItemType.ITEMS, "Deck of Every Card");
+      boolean allowedNormal =
+          StandardRequest.isAllowed(RestrictedItemType.ITEMS, "Deck of Every Card");
       boolean limited = KoLCharacter.getLimitMode().limitItem(ItemPool.DECK_OF_EVERY_CARD);
-      this.setShown((!bm || kf) && (have || nocards) && allowed && !limited);
+      this.setShown(
+          (!bm || kf) && (((haveNormal || nocards) && allowedNormal) || haveReplica) && !limited);
```

Comment:
I don't understand why "nocards" applies only to the normal Deck but not the replica Deck.

Reply:
There is a special message if you have already drawn all your cards.
If you have not drawn all your cards, you have to have a (usable) deck available in order to see the Daily Deed.

Seems like setShown could be
```(!bm || kf) && ((haveNormal && allowedNormal) || haveReplica) && !limited)```
although I don't see how you could ```haveNormal``` without ```allowedNormal```, unless you are under Standard restrictions and and pulled it anyway.  Can you do that?

Reply:
I don't think so -- you'd get a "this item is too old to use on this path" error.

Closer to the time, I think maybe you could hang out in Trendy (with the Deck) until the Deck expired, and then it would be out of standard. But you can't do that any more.